DocumentiPubblici/AnalisiDeiRequisiti/Sezioni/CasiUso/Venditore/GestioneOrdini.tex

```diff
@@ -1,12 +1,201 @@
 \UC{Visualizzazione riepilogo ordini in gestione}
-Il venditore vuole vedere gli ordini gestiti o da gestire.
+Il venditore vuole vedere gli ordini chiusi o da gestire.
 \begin{itemize}
     \item \textbf{Attori primari:} venditore;
-    \item \textbf{Precondizione:} il venditore da qualsiasi schermata in cui si trovi vuole visualizzare gli ordini da gestire/gestiti;
-    \item \textbf{Postcondizione:} Viene aperta la pagina di riepilogo ordini;
-    \item \textbf{Scenario principale:}
+    \item \textbf{Precondizione:} il venditore da qualsiasi schermata in cui si trovi vuole visualizzare gli ordini da gestire o già chiusi;
+    \item \textbf{Postcondizione:} il venditore vede tutti gli ordini a suo carico;
+    \item \textbf{Scenario principale:} il venditore vuole vedere tutti gli ordini a suo carico da gestire o già chiusi ed accede alla schermata di riepilogo ordini. Da qui può:
     \begin{itemize}
-        \item L'attore seleziona la funzione per visualizzare il riepilogo degli ordini in gestione;
-        \item Viene indirizzato alla schermata con tutti gli ordini a suo carico, per ogni ordine sono indicati i prodotti acquistati, la quantità e l'indirizzo di spedizione.
+    	\item (UC) - Visualizzazione ordini cronologicamente;
+        \item (UC) - Modificare lo stato di un ordine;
+        \item (UC) - Visualizzare i dettagli di un ordine;
+        \item (UC) - Ricerca di un ordine.
+        %\item (UC) - Contattare il cliente che ha effettuato un determinato ordine.
     \end{itemize}
+\end{itemize}
+
+\resetSubUC
+\subUC{Visualizzazione ordini cronologicamente}
```

Comment:
Io dividerei questo uc in due diversi (uno per ordine crescente e l'altro decrescente) e li metterei come casi principali e non come sotto uc visto che sono funzionalità a parte ed i sotto casi non possono introdurre funzionalità in più